code/datums/diseases/advance/symptoms/symptoms.dm

```diff
@@ -21,12 +21,20 @@ var/global/const/SYMPTOM_ACTIVATION_PROB = 5
 
 /datum/symptom/New()
 	var/list/S = list_symptoms
+
+	//Randomize Symptom Stats
+	stealth += rand(-1,1)

```

Comment:
The AddDisease proc in _MobProcs.dm copies any modifications made to the virus on infection; it was previously used so you could varedit viruses and keep the changes.